libraries/AC_PID/AC_P_1D.cpp

```diff
@@ -38,7 +37,7 @@ float AC_P_1D::update_all(float &target, float measurement)
     }
 
     // MIN(_Dxy_max, _D2xy_max / _kxy_P) limits the max accel to the point where max jerk is exceeded
-    return sqrt_controller(_error, _kp, _D1_max, _dt);
+    return sqrt_controller(_error, _kp, _D1_max, 0.0);
```

Comment:
What effect does hardcoding the dt argument here have?  I see this is done in both AC_P_1D and AC_P_2D.

Reply:
When using the square root controller for shaping we precisly follow this path to zero and get an occilation. However when using this controller in a PID loop the response is only aproximate and also noisy. So this dt is only important when using the square root for ideal shaping or comand model operations, not control operations.